st2client/st2client/utils/terminal.py

```diff
@@ -26,10 +27,20 @@
 ]
 
 
-def get_terminal_size(default=(80, 20)):
+def get_terminal_size(default=(80, 150)):
     """
     :return: (lines, cols)
     """
+    # Allow user to force terminal size using a environment variables
+    # E.g. ST2_CLI_FORCE_TERMINAL_SIZE=80,200 # lines, columns
```

Comment:
+1 for changing the default limits, but to control the terminal size, we can rely on existing logic, without implementing new `ST2_CLI_FORCE_TERMINAL_SIZE`: 

https://github.com/StackStorm/st2/blob/56f9cec9b9cfc973d4e03c062b9c9905d6fd34db/st2client/st2client/utils/terminal.py#L74-L78

Reply:
Yes, it will still rely on those two environment variables if other method fails, but with this new environment variable there is also a way to override those values (e.g. in case some other method return a value, but user wants to use a different one without changing xterm settings, etc.).